lib/ReactViews/ExplorerWindow/Tabs/DataCatalogTab.tsx

```diff
@@ -0,0 +1,94 @@
+import { runInAction } from "mobx";
+import { observer } from "mobx-react";
+import { useTranslation } from "react-i18next";
+import Box from "../../../Styled/Box";
+import DataCatalog from "../../DataCatalog/DataCatalog";
+import DataPreview from "../../Preview/DataPreview";
+import Breadcrumbs from "../../Search/Breadcrumbs";
+import SearchBox, { DEBOUNCE_INTERVAL } from "../../Search/SearchBox.jsx";
+import Styles from "./data-catalog-tab.scss";
+import { useViewState } from "../../Context";
+
+interface DataCatalogTabProps {
+  items?: unknown[];
+  searchPlaceholder?: string;
+  onActionButtonClicked?: () => void;
+}
+
+const DataCatalogTab = observer(function DataCatalogTab(
+  props: DataCatalogTabProps
+) {
+  const viewState = useViewState();
+  const { t } = useTranslation();
+
+  const {
+    searchState,
+    previewedItem: previewed,
+    breadcrumbsShown: showBreadcrumbs,
+    terria
+  } = viewState;
+
+  const searchPlaceholder =
+    props.searchPlaceholder || t("addData.searchPlaceholder");
+
+  const changeSearchText = (newText: string) => {
+    runInAction(() => {
+      viewState.searchState.catalogSearchText = newText;
```

Comment:
Should this use `searchState.X` instead of `viewState.searchState.X` just like the return statement a few lines below?